src/pages/SignUp/SignUp.module.css

```
@@ -2,74 +2,74 @@
   width: 100%;
   min-height: 100vh;
   padding: 24px;
+
   background-image: url(../../assets/SignUpBackground.png);
```

Stylelint (17.3.0)

[error] Unexpected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)